compose/galaxy-configurator/run.sh

```diff
@@ -120,8 +120,7 @@ else
   for conf in "${irida_configs[@]}"; do
     if [[ "$conf" == *.conf ]] ; then
       echo "Configuring $conf"
-      j2 --customize /customize.py --undefined -o "/tmp/$conf" "/templates/irida/$conf.j2" /base_config.yml
-
+      j2 --customize customize.py --undefined -o "/tmp/$conf" "templates/irida/$conf.j2" base_config.yml
```

Comment:
All the other uses of j2 use the absolute path forms - `/templates/irida/$conf.j2`, etc - why make this change to relative path names just for the IRIDA config? Also the [Dockerfile](https://github.com/combat-sars-cov-2/docker-galaxy-stable/blob/irida_support/compose/galaxy-configurator/Dockerfile#L6), the templates are specifically put in `/templates`.